config/output.ts

```diff
@@ -0,0 +1,13 @@
+import type { AstroUserConfig } from 'astro';
+
+/**
+ * Output mode for the Astro build process.
+ * - 'static': Pre-renders (SSG) all pages at build time into HTML files (default).
+ * - 'server': Use server-side rendering (SSR) for all pages by default, 
+ *    always outputting a server-rendered site.
+ * - The 'server' mode is used when preview mode is enabled, allowing dynamic SSR.
+ * Overriding individual pages to use SSR or SSG is possible by adding 
+ * `export const prerender = true // or false;` to a route.
+ * @see https://docs.astro.build/en/reference/configuration-reference/#output
+ */
+export const output = 'static' as const satisfies AstroUserConfig['output'];
```

Comment:
`as const` so that `datocmsRequest` has an inferred default.
`satisfies` so that the options remain clear.

See that the Typescript hint is the actual string 'static'

Reply:
When setting this to 'server', [this line ](https://github.com/voorhoede/head-start/pull/254/files#diff-536daf7ee02131ca906a17a9304fe8e0a7b53d135d47733667624d3c0ebeae57R61) breaks. Not exactly sure why and how to fix it though.

Reply:
I'll have a look!